README.md

```diff
@@ -148,6 +148,14 @@ IMPORTANT: This feature is very much *experimental*. Currently, the insertion
     paragraph is discarded. In the near future this feature will most likely be
     implemented on top of HTML insertion.
 
+##### Images [experimental]
+
+Add images to a document, the images data will be attach in the document.
+
+For an example see the test file: "image_test.rb"
```

Comment:
Would be nice to still have an inline example in the docs. Also it would be nice if the `image_test.rb` reference was a working link on GitHub.